components/breadcrumb/index.en-US.md

```diff
@@ -0,0 +1,32 @@
+---
+category: Components
+type: Navigation
+title: Breadcrumb
+---
+
+Breadcrumb displays the current page in the system hierarchy position, and can back to previous.
+
+## When to use
+
+- When the system has more than two hierarchy,
+- When you need to inform the user, "where are you",
+- When you need to back to previous.
```

Comment:
- When the system has more than two layers in a hierarchy.
- When you need to inform the user of where they are.
- When the user may need to navigate back to a higher level within the hierarchy.